File: cpp_solution/patterns/pattern11.cpp
//practice solution:
#include <iostream>
using namespace std;
int pattern11(int n){
    for(int i =1;i<=n+1;i++){
        for(int j= 1;j<i;j++){
            cout<<(i+j)%2<<" ";
            }
        cout<<endl;    
        }
}
int main(){
   int n=4;
   pattern11(n);
   return 0; 
    }

File: Python_solutions/patterns/pattern11.py
#Pattern 11

#Given an integer n. You need to recreate the pattern given below for any value of N. Let's say for N = 5, the pattern should look like as below:

#1 
#0 1 
#1 0 1 
#0 1 0 1 
#1 0 1 0 1

#Print the pattern in the function given to you.
#practice solution :
n=4
for i in range(1,n+1):
    for j in range(i):
        print((i +  j) % 2 , end=" ")
    print()

#for i in range(1, n+1):
#    num = i % 2  
#    for j in range(i):
#        print(num, end=" ")
#        num = 1 - num  
#    print()